src/test/java/org/openrewrite/java/migrate/guava/NoGuavaMapsNewTreeMapTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openrewrite.java.migrate.guava;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+
+class NoGuavaMapsNewTreeMapTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .recipe(new NoGuavaMapsNewTreeMap())
+          .parser(JavaParser.fromJavaVersion().classpath("guava"));
+    }
+
+    @DocumentExample
+    @Test
+    void replaceWithNewTreeMap() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.collect.*;
+              import java.util.Map;
+              
+              class Test {
+                  Map<Integer, Integer> cardinalsWorldSeries = Maps.newTreeMap();
+              }
+              """,
+            """
+              import java.util.Map;
+              import java.util.TreeMap;
+              
+              class Test {
+                  Map<Integer, Integer> cardinalsWorldSeries = new TreeMap<>();
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void replaceWithNewTreeMapWithMap() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import com.google.common.collect.*;
+              import java.util.Collections;
+              import java.util.Map;
+              import java.util.SortedMap;
+                            
```

Comment:
IntelliJ has an annoying bug here where the auto format introduces excess white space. I've found it only stops doing so when you completely remove any whitespace on those lines, all the way to the beginning of the line.
```suggestion

```

On my machine I've enabled my IDE to show whitespace, such that these issues and tabs stand out more, I suggest you do the same.
![image](https://github.com/user-attachments/assets/4af89370-aa81-49c7-95bc-0aad7f379ad2)

And then if you use that `F2` repeately before a commit you'll see prompts like this one to fix it everywhere.
![image](https://github.com/user-attachments/assets/5c4313f6-a90b-4467-bee3-c96ef9e2650e)

Reply:
Must have missed it in this case, I have enabled show whitespaces now :)